### PR TITLE
[trans] id_ID

### PR DESCRIPTION
**Pull request by suabahasa**

### CODE REVIEW

**Comment by jasonvarga** [comment]
Thank you for this PR. Are you planning on adding the rest of the translations?

**Comment by ebeauchamps** [comment]
Indonesian..interesting. Did you try following [those guidelines](https://statamic.dev/cp-translations#content) to help you out? language code should be "id"...but you already knows that!

**Comment by suabahasa** [comment]
Yes, I will commit the rest translation progressively.

**Comment by jasonvarga** [comment]
Great! Just mark the PR as ready for review when you're done.

**Comment by suabahasa** [comment]
I have a little knowledge about using git, is it common that this pull request include commit from [182164f](https://github.com/statamic/cms/pull/2426/commits/182164fa30ff1f18b8405796ee409e7068709d77) and [b919007](https://github.com/statamic/cms/pull/2426/commits/b919007091a78bcb8695aa8f06e29d9bb08e9e66) ?

**Review by ebeauchamps — COMMENTED** [review]
Hello! you also have to generate an   id.json   file. You can also duplicate any other  xx.json  file found in vendor/statamic/cms/resources/lang, change the duplicate name to id.json and translate all English keys in Indonesian

**Comment by suabahasa** [comment]
> 
> 
> Hello! you also have to generate an id.json file. You can also duplicate any other xx.json file found in vendor/statamic/cms/resources/lang, change the duplicate name to id.json and translate all English keys in Indonesian.

Hi. Thanks for guide.

File: resources/lang/id.json [hunk 1]
```diff
@@ -0,0 +1,796 @@
...
+    "You are not authorized to delete this taxonomy.": "",
+    "You are not authorized to edit this collection.": "",
+    "You are not authorized to edit this taxonomy.": "",
+    "You are not authorized to run this action.": "",
+    "You are not authorized to scaffold resources.": "",
+    "You are not authorized to view collections.": "",
+    "You are not authorized to view navs.": "",
+    "You are not authorized to view this collection.": "",
+    "Your Favorites": "",
+    "Your working copy will be replaced by the contents of this revision.": ""
```
Comment (jasonvarga) [inline resources/lang/id.json:795]: These are still empty.

**Review by jasonvarga — CHANGES_REQUESTED** [review]

File: resources/lang/id.json (hunk 1 above)
Comment (suabahasa) [reply resources/lang/id.json:795]: Yes, will commit the rest soon

**Comment by damcclean** [comment]
@suabahasa It looks like the values are no longer blank. Is this pull request ready for review?

**Comment by suabahasa** [comment]
@damcclean there are left about 20% of string (id.json) not yet translated. A bit busy this week, will commit the 20% the rest and submit it for review

**Comment by damcclean** [comment]
> @damcclean there are left about 20% of string (id.json) not yet translated. A bit busy this week, will commit the 20% the rest and submit it for review

No rush, just wanted to know how it's going 😄

**Comment by jasonvarga** [comment]
If you don't mind, I'm going to use Google Translate to fill in the missing lines shortly just so I can get this merged. You can feel free to review it after I do that. Thank you for the progress!

**Comment by suabahasa** [comment]
Sure, I will happy to review the translation. Thanks

**Comment by jasonvarga** [comment]
Thank you again. I've filled in the translations in dfbc54c9

**Pull request merged**